debian/gen_bootloader_postinst_preinst.sh

```diff
@@ -79,6 +79,13 @@ if ! /bin/grep -Eq "^blacklist 8192cu" /etc/modprobe.d/blacklist-rtl8192cu.conf
   echo "blacklist 8192cu" >> /etc/modprobe.d/blacklist-rtl8192cu.conf
 fi
 /bin/sed -i -e '/^blacklist rtl8192cu/s/^/#/' /etc/modprobe.d/blacklist-rtl8192cu.conf
+
+# Remove deprecated "elevator=deadline" from the cmdline.txt
+# We will only do anything if we are certain that the user has not modfified the
+# relevant part of the cmdline.
+if ! /bin/grep -Fq "fsck.repair=yes elevator=deadline rootwait" /boot/cmdline.txt ; then
```

Comment:
On my devices in /boot/cmdline.txt, they are "elevator=deadline fsck.repair=yes rootwait"

Reply:
Same here:

```
pi@RevPi31273:~ $ cat /boot/cmdline.txt
dwc_otg.lpm_enable=0 console=tty1 root=/dev/mmcblk0p2 rootfstype=ext4 elevator=deadline fsck.repair=yes rootwait nosplash plymouth.ignore-serial-consoles
```

```
pi@RevPi6351:~ $ cat /boot/cmdline.txt
dwc_otg.lpm_enable=0 console=tty1 root=/dev/mmcblk0p2 rootfstype=ext4 elevator=deadline fsck.repair=yes rootwait nosplash plymouth.ignore-serial-consoles
```

Reply:
Good catch. I got mixed up in the test files I created.